### PR TITLE
Bolo for Centos7 to use systemd

### PR DESCRIPTION
**Pull request by GrayTShirt**
Add support for systemd for Centos7 and greater.

### CODE REVIEW

**Comment by GrayTShirt** [comment]
I feel dirty...

File: bolo/rpm.spec [hunk 1]
```diff
@@ -223,28 +306,46 @@ This package provides the postgres subscriber component for bolo.
...
 %{_sbindir}/bolo2pg
-%{_initrddir}/bolo2pg
+%if 0%{?rhel} >= 7
+	%{_unitdir}/bolo2pg.service
+%else
+	%{_initrddir}/bolo2pg
+%endif
 %{_mandir}/man8/bolo2pg.8.gz
-%doc %{_datadir}/bolo
+%doc %{_datadir}/bolo/schema/pg.sql
```
Comment (jhunt) [inline bolo/rpm.spec]: This change looks out of place for a systemd init changeover...

File: bolo/rpm.spec [hunk 2]
```diff
@@ -48,12 +48,22 @@ rm -f $RPM_BUILD_ROOT%{_bindir}/bolo_nsca
...
-install -m 0755 -D redhat/init.d/bolo2pg    $RPM_BUILD_ROOT%{_initrddir}/bolo2pg
-install -m 0755 -D redhat/init.d/bolo2meta  $RPM_BUILD_ROOT%{_initrddir}/bolo2meta
+%if 0%{?rhel} >= 7
+	install -m 0755 -D /srv/systemd/bolo.service        $RPM_BUILD_ROOT%{_unitdir}/bolo.service
+	install -m 0755 -D /srv/systemd/bolo-cache.service  $RPM_BUILD_ROOT%{_unitdir}/bolo-cache.service
+	install -m 0755 -D /srv/systemd/bolo2meta.service   $RPM_BUILD_ROOT%{_unitdir}/bolo2meta.service
+	install -m 0755 -D /srv/systemd/bolo2pg.service     $RPM_BUILD_ROOT%{_unitdir}/bolo2pg.service
+	install -m 0755 -D /srv/systemd/bolo2redis.service  $RPM_BUILD_ROOT%{_unitdir}/bolo2redis.service
+	install -m 0755 -D /srv/systemd/bolo2rrd.service    $RPM_BUILD_ROOT%{_unitdir}/bolo2rrd.service
+	install -m 0755 -D /srv/systemd/dbolo.service       $RPM_BUILD_ROOT%{_unitdir}/dbolo.service
```
Comment (jhunt) [inline bolo/rpm.spec]: The `/srv/systemd` paths should be relative, like the `redhat/init.d` paths below.

File: bolo/systemd/bolo-cache.service [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Bolo Caching Daemon
+Documentation=man:bolo-cache(1)
+After=network.target
+
+[Service]
+EnvironmentFile=/etc/sysconfig/bolo-cache
```
Comment (jhunt) [inline bolo/systemd/bolo-cache.service]: If/when debuntu moves to systemd, these files probably won't be re-usable.  Can you move them from `bolo/systemd` (in-repo) to `bolo/redhat/systemd`?

Comment (GrayTShirt) [inline bolo/systemd/bolo-cache.service]: I took the liberty of moving the specfile into redhat/ also

**Pull request merged**